Unit tests are failing on **Python 2.6**


```
self = <units.modules.network.ios.test_ios_banner.TestIosBannerModule testMethod=test_ios_banner_create>

    def test_ios_banner_create(self):
        for banner_type in ('login', 'motd', 'exec', 'incoming', 'slip-ppp'):
            set_module_args(dict(banner=banner_type, text='test\nbanner\nstring'))
>           commands = ['banner {} @\ntest\nbanner\nstring\n@'.format(banner_type)]
E           ValueError: zero length field name in format

test/units/modules/network/ios/test_ios_banner.py:51: ValueError
```

--- a/lib/ansible/modules/network/ios/ios_banner.py
+++ b/lib/ansible/modules/network/ios/ios_banner.py
@@ -37,8 +37,8 @@ notes:
 options:
   banner:
     description:
-      - Specifies which banner that should be
-        configured on the remote device.
+      - Specifies which banner should be configured on the remote device.
+        In Ansible 2.4 and earlier only I(login) and I(motd) were supported.
     required: true
     default: null
     choices: ['login', 'motd', 'exec', 'incoming', 'slip-ppp']

--- a/lib/ansible/modules/network/ios/ios_banner.py
+++ b/lib/ansible/modules/network/ios/ios_banner.py
@@ -41,7 +41,7 @@ options:
         configured on the remote device.
     required: true
     default: null
-    choices: ['login', 'motd']
+    choices: ['login', 'motd', 'exec', 'incoming', 'slip-ppp']
   text:
     description:
       - The banner text that should be
@@ -151,7 +151,7 @@ def main():
     """ main entry point for module execution
     """
     argument_spec = dict(
-        banner=dict(required=True, choices=['login', 'motd']),
+        banner=dict(required=True, choices=['login', 'motd', 'exec', 'incoming', 'slip-ppp']),
         text=dict(),
         state=dict(default='present', choices=['present', 'absent'])
     )

--- a/test/units/modules/network/ios/test_ios_banner.py
+++ b/test/units/modules/network/ios/test_ios_banner.py
@@ -48,7 +48,7 @@ class TestIosBannerModule(TestIosModule):
     def test_ios_banner_create(self):
         for banner_type in ('login', 'motd', 'exec', 'incoming', 'slip-ppp'):
             set_module_args(dict(banner=banner_type, text='test\nbanner\nstring'))
-            commands = ['banner {} @\ntest\nbanner\nstring\n@'.format(banner_type)]
+            commands = ['banner {0} @\ntest\nbanner\nstring\n@'.format(banner_type)]
             self.execute_module(changed=True, commands=commands)
 
     def test_ios_banner_remove(self):

--- a/test/units/modules/network/ios/test_ios_banner.py
+++ b/test/units/modules/network/ios/test_ios_banner.py
@@ -46,9 +46,10 @@ class TestIosBannerModule(TestIosModule):
         self.load_config.return_value = dict(diff=None, session='session')
 
     def test_ios_banner_create(self):
-        set_module_args(dict(banner='login', text='test\nbanner\nstring'))
-        commands = ['banner login @\ntest\nbanner\nstring\n@']
-        self.execute_module(changed=True, commands=commands)
+        for banner_type in ('login', 'motd', 'exec', 'incoming', 'slip-ppp'):
+            set_module_args(dict(banner=banner_type, text='test\nbanner\nstring'))
+            commands = ['banner {} @\ntest\nbanner\nstring\n@'.format(banner_type)]
+            self.execute_module(changed=True, commands=commands)
 
     def test_ios_banner_remove(self):
         set_module_args(dict(banner='login', state='absent'))

